components/mdns/docs/en/index.rst

```diff
@@ -0,0 +1,189 @@
+mDNS Service
+============
+:link_to_translation:`zh_CN:[中文]`
```

Comment:
Just a note, looks like the [Chinese documentation](https://docs.espressif.com/projects/esp-idf/zh_CN/latest/esp32/api-reference/protocols/mdns.html) got lost while moving from IDF.